docset/winserver2022-ps/configci/New-CIPolicyRule.md

```diff
@@ -192,6 +197,48 @@ attributes     : {[AppIDs, ], [MinimumFileVersion, 0.0.0.0], [FilePath, .\temp\C
 
 This command generates a filepath rule for the specific path verbatim string. This will allow anything in the parent folder. 
 
+### Example 5: Create a policy rule for a packaged application and its dependencies
+```
+PS C:\> $package = Get-AppxPackage -Name *Microsoft.Whiteboard*
+PS C:\> $package_dependencies = $package.Dependencies
+
+PS C:\> $package_rule = New-CIPolicyRule -Package $package
+PS C:\> $package_rule += New-CIPolicyRule -Package $dependency[0] # repeat for all dependencies in array
+
+PS C:\> $package_rule
+
+
+Name           : Microsoft.Whiteboard_8wekyb3d8bbwe FileRule
```

Comment:
Can you wrap the output in ```output?

Reply:
This is not aligned with the rest of the topic articles but I have added in latest commit (https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2483/commits/05ecd0c211df5df115e5821224eb3b87322e6a43)